bibliopixel/__init__.py

```diff
@@ -15,12 +16,23 @@
     from . layout import matrix_drawing as matrix
     from . layout.geometry.rotation import Rotation
 
+__version__ = '3.4.18'
+__status__ = 'Development'
```

Comment:
So I was unable to find out any documentation on this `__status__` variable in Python...?

Reply:
These dunders are for module metadata.
`\_\_status\_\_` comes from some older work that used epydoc.
I've been using this style for module metadata for a while. I'm not convinced it's the /right/ way, but I'm also unsure what the /right/ way is.

Some reading regarding this:
https://web.archive.org/web/20111010053227/http://jaynes.colorado.edu/PythonGuidelines.html#module_formatting
http://epydoc.sourceforge.net/manual-fields.html#module-metadata-variables

I couldn't find much information about this right-away from the Python documentation.
Pep8 brushes on `\_\_version\_\_` and `\_\_author\_\_`, but not much else.